src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseCountProperly.cs

```diff
@@ -596,12 +595,13 @@ private static bool IsRightCountComparison(IBinaryOperation binaryOperation, out
             return true;
         }
 
-        private static bool TypeContainsVisibleProperty(OperationAnalysisContext context, ITypeSymbol type, string propertyName, SpecialType propertyType, out ISymbol? propertySymbol)
-            => TypeContainsVisibleProperty(context, type, propertyName, propertyType, propertyType, out propertySymbol);
+        private static bool TypeContainsVisiblePropertyWithoutCircularity(OperationAnalysisContext context, IOperation operation, ITypeSymbol type, string propertyName, SpecialType propertyType, [NotNullWhen(true)] out ISymbol? propertySymbol)
+            => TypeContainsVisiblePropertyWithoutCircularity(context, operation, type, propertyName, propertyType, propertyType, out propertySymbol);
 
-        private static bool TypeContainsVisibleProperty(OperationAnalysisContext context, ITypeSymbol type, string propertyName, SpecialType lowerBound, SpecialType upperBound, out ISymbol? propertySymbol)
+        private static bool TypeContainsVisiblePropertyWithoutCircularity(OperationAnalysisContext context, IOperation operation, ITypeSymbol type, string propertyName, SpecialType lowerBound, SpecialType upperBound, [NotNullWhen(true)] out ISymbol? propertySymbol)
```

Comment:
[cosmetic] Could we wrap this long line?
```suggestion
        private static bool TypeContainsVisiblePropertyWithoutCircularity(OperationAnalysisContext context, IOperation operation, ITypeSymbol type,
        	string propertyName, SpecialType lowerBound, SpecialType upperBound, [NotNullWhen(true)] out ISymbol? propertySymbol)
```